src/libraries/System.Threading.Tasks/tests/System.Runtime.CompilerServices/TaskAwaiterTests.cs

```diff
@@ -561,6 +561,56 @@ public static IEnumerable<object[]> CanceledTasksAndExpectedCancellationExceptio
             yield return new object[] { LineNumber(), atmb.Task, oce };
         }
 
+        [Theory]
+        [InlineData((ConfigureAwaitOptions)0x8)]
+        public void ConfigureAwaitOptions_Invalid(ConfigureAwaitOptions options)
+        {
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("options", () => Task.CompletedTask.ConfigureAwait(options));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("options", () => Task.FromResult(true).ConfigureAwait(options));
+        }
+
+        [Fact]
+        public void ConfigureAwaitOptions_SuppressThrowingUnsupportedOnGenericTask()
+        {
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("options", () => Task.FromResult(true).ConfigureAwait(ConfigureAwaitOptions.SuppressThrowing));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("options", () => Task.FromResult(true).ConfigureAwait((ConfigureAwaitOptions)0x7));
```

Comment:
Don't recall the actual values of the enum, but presumably `0x7` contains the bits of `SuppressThrowing`?